scripts/sync_wiki_item_langs.py

```
@@ -0,0 +1,437 @@
+    batches = find_latest_batches()
+    for code, batch in batches.items():
+        if code != "zh-Hant":
+            continue
+        items = load_items(batch)
+        for iid, (name, aid) in items.items():
+            if aid:
+                aid_to_iid.setdefault(aid, iid)
+
+    matched = {}
+    unmatched = []
+    for sl_iid, (name, aid) in zhcn_items.items():
```

Ruff (0.16.0)

[warning] Loop control variable `sl_iid` not used within loop body

Rename unused `sl_iid` to `_sl_iid`

(B007)